src/java/com/sarality/app/view/dialog/CustomViewDialogFragment.java

```diff
@@ -33,14 +33,21 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
   public void onActivityCreated(Bundle savedInstanceState) {
     super.onActivityCreated(savedInstanceState);
     dataSource = createDataSource();
-    if (dataSource == null) {
-      render(null);
-    } else {
+    if (dataSource != null) {
```

Comment:
Because of the loaderId problem ,  by the time the consume is called, the app usually crashes. Because the dialog expects a different data type but the loader is giving different data type. 
The consume has to be blocked.